cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,37 @@
+# https://gitlab.kitware.com/cmake/community/-/wikis/FAQ#can-i-do-make-uninstall-with-cmake
```

Comment:
I also think this isn't required for this pr

Reply:
Yes, removed